source/iterator/position.go

```diff
@@ -0,0 +1,66 @@
+// Copyright © 2023 Meroxa, Inc. & Yalantis
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package iterator
+
+import (
+	"encoding/json"
+	"fmt"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+// positionMode defines the [position] mode.
+type positionMode string
+
+// The available position modes are listed below.
+const (
+	modeSnapshot positionMode = "snapshot"
+	modeCDC      positionMode = "cdc"
+)
+
+// position is an iterator position.
+type position struct {
+	Mode positionMode `json:"mode"`
+	// Element is a value of the last processed element by the snapshot capture.
+	// This value is used if the mode is snapshot.
+	Element any `json:"element"`
```

Comment:
What do you think about rename it to `Value` or `LastProccesedValue` ?

Reply:
Yeah, sure, `LastProcessedValue` would work fine. I've renamed it.